p4include/p4d2model_bmss_meta.map

```diff
@@ -0,0 +1,29 @@
+# This file maps p4d2 metadata to the bmv2 simple switch standard and intrinsic metadata
+
+# parser
+parser_input_t.ingress_port       -> standard_metadata.ingress_port
```

Comment:
Why do you map <type>.<field> into <instance>.<field>? In other words why this mapping instead of 
parser_input_t.ingress_port -> stadnard_metadata_t.ingress_port